app_flair.py

```diff
@@ -68,18 +81,22 @@ def route_predict_ner():
 
 @app.route("/ner/train", methods=["POST"])
 def route_train_ner():
+    # Deal with the ner training request
+    # Use the lock to make sure that only one training is in the process for each time
+    # The training is in a background thread
     if lock_ner_train.acquire(block=False):
         json_data = request.get_json()
         train, dev = parse_ner_train_request(json_data)
-        t1 = threading.Thread(target=train_ner, args=(train, dev))
+        t1 = threading.Thread(target=flair_train_ner, args=(train, dev))
         t1.start()
-        return ('', 204)
+        return "Successful training", 204
```

Comment:
You can use `http.HTTPStatus`, e.g by returning ` HTTPStatus.TOO_MANY_REQUESTS`. Same for the other status codes.